docs/index.md

```diff
@@ -186,12 +186,15 @@ Examples of websites built using MarkBind:
 
 MarkBind is a project based in the [National University of Singapore, School of Computing](http://www.comp.nus.edu.sg/), and is funded by a grant from [NUS Center for Development of Teaching and Learning](http://www.cdtl.nus.edu.sg/).
 
-* [**Aaron Chong Jun Hao**](https://github.com/acjh): Current _Team Lead_ since Aug 2017
+* [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ since Jul 2018, _Team Lead_ for Aug 2017 - Jun 2018
+* [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Project Member_ since May 2018
 * [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Project Member_ since Jan 2018
 * [**Damith C. Rajapakse**](https://github.com/damithch): _Project Mentor_ since Aug 2016
 * [**Daniel Berzin Chua Yuan Siang**](https://github.com/danielbrzn): _Project Member_ since Jan 2018
+* [**Jia Zhixin**](https://github.com/nusjzx): _Project Member_ since May 2018
 * [**Jiang Sheng**](https://github.com/Gisonrg): _Founding Member_ and _Team Lead_ for Aug 2016 - Jul 2017
 * [**Rachael Sim Hwee Ling**](https://github.com/rachx): _Project Member_ since Jan 2018
+* [**Tan Wang Leng**](https://github.com/yamgent): Current _Team Lead_ since Jul 2018, _Project Member_ for May 2018 - Jun 2018
```

Comment:
We'll leave things like this for now. We can separate out current and past members when the list is longer and when some members have been inactive for an extended period.